store/postgres/migrations/2025-10-30-094807-0000_table_stats_view/up.sql

```diff
@@ -0,0 +1,35 @@
+CREATE MATERIALIZED VIEW info.table_stats AS
```

Comment:
It's a little unfortunate that we already have a `subgraphs.table_stats` which has a completely different meaning. It might be good to use a different name here, maybe `entity_version_stats`, though more than happy with something else